docker/tests/sidecar/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gcr.io/cicd-246518/dotcms-curl
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Always tag the version of an image explicitly](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5442148)